microsoft-edge/progressive-web-apps-chromium/how-to/index.md

```diff
@@ -1,113 +1,139 @@
+<!-- link is placed here per https://github.com/MicrosoftDocs/edge-developer/issues/1974 -->
+This guide is targeted at web developers who want to learn to build PWAs. To learn more about installing and running PWAs, see [Installing a PWA](../ux.md#installing-a-pwa) in _Use Progressive Web Apps in Microsoft Edge_.
 
-> [!TIP]
-> You can use [PWABuilder](https://www.pwabuilder.com) to create a new PWA, enhance your existing PWA, or package your PWA for app stores.
+In this guide, you first learn how PWAs work, then create your first simple PWA, which will be a temperature converter app, and then learn more about how to make great PWAs.
 
-<!-- link is placed here per https://github.com/MicrosoftDocs/edge-developer/issues/1974 -->
-To install or run a PWA on Windows, see [Installing a PWA](../ux.md#installing-a-pwa) in _The user experience of PWAs_.
+You can find the final source code of the app you will be building in this guide on the [PWA getting started demo app repository](https://github.com/MicrosoftEdge/Demos/tree/main/pwa-getting-started).
 
 
 <!-- ====================================================================== -->
 ## Prerequisites
 
 *   Install [Visual Studio Code](https://code.visualstudio.com) to edit your PWA source code.
 *   Install [Node.js](https://nodejs.org) to use it as your local web server.
+*   Working knowledge of HTML, CSS, and JavaScript is also a plus.
 
 
 <!-- ====================================================================== -->
-## Creating a basic website
+## The architecture of a PWA
 
-In this step, you will create a simple HTML page to create a basic website, and will start a local web server to access the website in your browser.
+Progressive Web Apps are written using the programming languages of the web: HTML, CSS, and JavaScript, and are distributed to your users by using web servers.
 
-1. Create a new directory called `MySamplePWA` on your computer.
+To make your app available to users, you deploy it on a web server that's accessible via HTTPS. Your server contains:
 
-1. Open Visual Studio Code, select **File** > **Open Folder** and then select the `MySamplePWA` directory to open your new project.
+* **Back-end code**: the endpoints needed by your app, when connected to the internet, to retrieve dynamic content that may be stored in a database on your server.
+* **Front-end code**: the resources needed for the app to be installed on the user's device, such as HTML, CSS, and JavaScript code.
 
-1. Create a new file in the project by pressing `Ctrl+N`, add the following content, and save the file as `index.html`:
+Your back-end code can use the server-side languages of your choice such as ASP.NET, Java, Node.js, or PHP. Note, however, that server-side endpoints may not even be required depending on the app your are building. The PWA you create in this tutorial does not have any server-side code because it exclusively runs on the device it's installed on and doesn't need any server-side data.
```

Comment:
```suggestion
Your back-end code can use the server-side languages of your choice such as ASP.NET, Java, Node.js, or PHP. Note, however, that server-side endpoints may not even be required, depending on the app you're building. The PWA that you create in this tutorial doesn't have any server-side code, because the app exclusively runs on the device it's installed on, and doesn't need any server-side data.
```